Apps/AdminWebClient/src/AdminWebClient.csproj

```diff
@@ -78,7 +76,7 @@
     <!-- Include the newly-built files in the publish output -->
     <ItemGroup>
       <DistFiles Include="$(SpaRoot)dist\**" />
-      <ResolvedFileToPublish Include="@(DistFiles-&gt;'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
+      <ResolvedFileToPublish Include="@(DistFiles->'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
```

Comment:
Can this '>' symbol cause issue to our devs who use mac?

Reply:
VS on Windows encodes it this way while VS on Mac does the &gt way.